pyproject.toml

```diff
@@ -168,6 +179,9 @@ max-complexity = 10
     "F401", # unused import
     "F403", # star imports
 ]
+"models.py" = [
+    "UP007", # non-pep604-annotation - pydantic models use annotations at runtime
+]
```

Comment:
Not super happy about `UP007` not respecting the `runtime-evlated-base-classes`
I will raise this issue on https://github.com/astral-sh/ruff